src/IO/HTTPCommon.cpp

```diff
@@ -120,7 +120,7 @@ namespace
                 session->setProxyHost(proxy_host);
                 session->setProxyPort(proxy_port);
 
-#if !defined(ARCADIA_BUILD)
+#if defined(POCO_CLICKHOUSE_PATCH)
```

Comment:
Can someone take a look at yandex check failures? Maybe arcadia also defines `POCO_CLICKHOUSE_PATCH`?

I guess this should be rewritten like this:

```suggestion
#if !defined(ARCADIA_BUILD) && defined(POCO_CLICKHOUSE_PATCH)
```

Reply:
Yes, arcadia has it.